scripts/devenv-builder/create-vm.sh

```diff
@@ -52,7 +52,7 @@ if [ "${SWAPSIZE}" -eq 0 ] ; then
     sed -i "s;^part swap;#part swap;" ${KICKSTART_FILE}
 fi
 
-sudo dnf install -y libvirt virt-manager virt-install virt-viewer libvirt-client qemu-kvm qemu-img
+sudo dnf install -y libvirt virt-manager virt-install virt-viewer libvirt-client qemu-kvm qemu-img sshpass
```

Comment:
Please, also update the https://github.com/openshift/microshift/blob/main/docs/devenv_setup.md#creating-vm doc.

Also, we have another `create-vm.sh` script+doc for image building, so we may want to make it consistent. See https://github.com/openshift/microshift/blob/main/scripts/image-builder/create-vm.sh

Reply:
Sounds good, updated!